shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/RealtimePatterns.kt

```diff
@@ -33,13 +35,15 @@ sealed class RealtimePatterns : Comparable<RealtimePatterns> {
         override val patterns: List<RoutePattern>,
         override val upcomingTrips: List<UpcomingTrip>? = null,
         override val alertsHere: List<Alert>? = null,
+        override val hasSchedulesToday: Boolean,
```

Comment:
For tests, it would be convenient if `hasSchedulesToday` defaulted to `true` - that's why `upcomingTrips` and `alertsHere` default to `null` - and if `hasSchedulesToday` doesn't have a default and isn't moved before `upcomingTrips` and `alertsHere` then the defaults for `upcomingTrips` and `alertsHere` will never be usable and should be removed. A lot of the files touched here are just catching up to the lack of defaults here.

Reply:
Yeah that's fair. It seemed wrong to assume that there were schedules by default, but that's what we're doing anyway before schedules load, so I'm fine with just having the default, and it definitely cuts down on the annoying test changes.